packages/tupaia-web/src/features/Map/utils/useActiveMapOverlayReport.ts

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
I renamed this file to highlight that this hook gets the data for the selected / active map overlay and to differentiate it from the generic useMapOverlay hook